frontend/src/router/index.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  createRouter, createWebHashHistory, Router, RouteRecordRaw,
+} from 'vue-router';
+import Home from '../views/c-home.vue';
+
+const routes: Array<RouteRecordRaw> = [
+  {
+    path: '/',
+    component: Home,
+  },
+];
+
+const router: Router = createRouter({
+  history: createWebHashHistory(),
```

Comment:
We discussed it in class but I think it would be beneficial if you could elaborate on the reason you chose the hash based history mode instead of the default one

Docs for reference: https://router.vuejs.org/guide/essentials/history-mode.html#hash-mode

Reply:
Sure, I previously write it in #1894, but I think it is a good idea to transfer it here since I am splitting up the PR. I have added it to the description, thanks for the suggestion.